Makefile

```diff
@@ -87,7 +87,7 @@ clear-cache: ## Clear cache prod: make-clear-cache env=[dev|prod|test]
 
 cc: clear-cache
 
-clear-pool: ## Clear cache pool
+clear-pool: ## Clear cache pool: make clear-pool pool=[pool_name]
```

Comment:
Ajout exemple, suite à la discussion sur les cache avec @emilschn :-D
Pour en savoir plus https://symfony.com/doc/current/cache.html#clearing-the-cache